sumo/cli/dosplot.py

```diff
@@ -174,11 +174,6 @@ def dosplot(filename=None, code='vasp', prefix=None, directory=None,
                              total_only)
 
     elif code.lower() == 'castep':
-        for arg in sumo.io.castep.unsupported_dosplot_args:
```

Comment:
Maybe we should add some hook function so whenever a TDOS is being plotted we warn the user about irrelevant arguments they asked for. That's not CASTEP-specific and should maybe be a follow-up rather than part of this PR? In the meantime it does make sense to cut this as it's no longer a CASTEP issue.

Reply:
Probably better for another PR to address it once for all? I think it is better to be addressed at a code agnostic level.

Reply:
Created issue #26 to record this